system/datatypes/peer_reviewed_article.php

```diff
@@ -1108,6 +1231,9 @@ private function createCommentForm($type, array $values = [], HyphaDomElement $d
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
 				<strong><label for="[[field-name-email]]">[[email]]</label></strong> <input type="text" id="[[field-name-email]]" name="[[field-name-email]]" />
 			</div>
+			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
+				<strong><label for="[[field-name-subscribe]]">[[subscribe]]</label></strong> <input value="[[field-value-subscribe-me]]" type="checkbox" id="[[field-name-subscribe]]" name="[[field-name-subscribe]]" /> [[info-button-subscribe]]
+			</div>
```

Comment:
I haven't checked, but this looks like the subscribe checkbox is *always* shown, even for logged in users that are automatically subscribed currently (and do not need to enter or confirm their e-mail). Or am I mistaken?

Reply:
This part of the code is only used for non-users.